courses/free-courses-bn.md

```diff
@@ -210,6 +211,8 @@
 * [Next Js Bangla Tutorial Series](https://www.youtube.com/playlist?list=PLwMeE9AWeV59vbQSIArd0-sNB9FPxlWSp) - dSkill
 * [next js bangla(বাংলা) tutorial](https://www.youtube.com/playlist?list=PLQvUYGXiwrskS_C3MOeW0rOVB5Ny2MCR2) - Faazle Rabbi
 * [Next js tutorial in bangla](https://www.youtube.com/playlist?list=PLkmCJMhveta1PihBgW4MpYxFlyzdSjlNC) - Sabeek Bin Sayeed
+* [Next JS  14 Full Course in bangla](https://www.youtube.com/watch?v=XnIm8mb7WOw&list=PLNMnAEqLBwmqt3wGvc_JMqFobBZBcpv8c) - Hablu Programmer
```

Comment:
This needs to be alphabetical order, based on the tittle you might need to add it in the beginning and not the end.